snapcraft.yaml

```diff
@@ -0,0 +1,41 @@
+name: subiquity
+version: "0.0.32"
+summary: Ubuntu installer
+description: The Ubuntu server installer
+confinement: classic
+
+apps:
+  subiquity:
+    command: usr/bin/subiquity
+  console-conf:
```

Comment:
I don't think we should have an app for console-conf (probert is just about ok :-))